a2z.py

```diff
@@ -0,0 +1,447 @@
+#!/usr/bin/env python3
+
+"""
+
+Copyright (C) 2020 Vanessa Sochat.
+
+This Source Code Form is subject to the terms of the
+Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
why not apache 2?

Reply:
Why not Mozilla 2?

Reply:
Because MPL is copyleft, that would limit adoption/applicability of this library happen someone wants to create a derived product.